src/routeToRegExp.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+/* global routeToRegExp: true */
+
+/**
+ * @param path {string} path
+ * @param opts {Object} options
+ * @return {?Object}
+ *
+ * @description
+ * Normalizes the given path, returning a regular expression
+ * and the original path.
+ *
+ * Inspired by pathRexp in visionmedia/express/lib/utils.js.
+ */
+function routeToRegExp(path, opts) {
+  if (!angular.isString(path)) path = '';
```

Comment:
AFAICT, this function would never be called with something that is not a string (if people followed the docs :grin:).
I suggest we remove this line (and fail hard if `path` is not a string, so that we can detect issues quicker), instead of covering it up with a route that does not correspond to the passed in `path`.